ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRModeControls.java

```diff
@@ -18,6 +18,9 @@ public RDXVRModeControls(RDXVRModeManager vrModeManager)
    {
       this.vrModeManager = vrModeManager;
       baseUI = RDXBaseUI.getInstance();
+
+      // Optionally accessible as a normal panel
+      baseUI.getImGuiPanelManager().addPanel(PANEL_NAME, this::render);
```

Comment:
Important when you don't have VR and need to access the VR panel. I tested this and even with both panels open I saw no issues. Makes sense because you've got to have the 3D situated one instantiated too.